android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -229,11 +229,44 @@ function deregisterModel(modelID) {
   keyman.removeModel(modelID);
 }
 
-function enableSuggestions(model, mayPredict, mayCorrect) {
+function enableSuggestions(model, mayPredict, maySuggest) {
```

Comment:
Why do we still have `mayPredict` if the app now loads it all into `maySuggest`? Shouldn't this be:

```suggestion
function enableSuggestions(model, suggestionType) {
```